docs/src/locators.md

```diff
@@ -673,30 +661,30 @@ var texts = await rows.EvaluateAllAsync("list => list.map(element => element.tex
 
 If you have a list of identical elements, and the only way to distinguish between them is the order, you can choose a specific element from a list with [`method: Locator.first`], [`method: Locator.last`] or [`method: Locator.nth`].
 
-However, use these methods with caution. Often times, the page might change, and locator will point to a completely different element from the one you expected. Instead, try to come up with a unique locator that will pass the [strictness criteria](#strictness).
-
 For example, to click the third item in the list of products:
 
 ```js
-await page.locator('data-test-id=product-card').nth(3).click();
+await page.getByTestId('product-card').nth(3).click();
 ```
 
 ```java
-page.locator("data-test-id=product-card").nth(3).click();
+page.getByTestId("product-card").nth(3).click();
 ```
 
 ```python async
-await page.locator("data-test-id=product-card").nth(3).click()
+await page.get_by_test_id("product-card").nth(3).click()
 ```
 
 ```python sync
-page.locator("data-test-id=product-card").nth(3).click()
+page.get_by_test_id("product-card").nth(3).click()
 ```
 
 ```csharp
-await page.Locator("data-test-id=product-card").Nth(3).ClickAsync();
+await page.GetByTestId("product-card").Nth(3).ClickAsync();
 ```
 
+However, use these methods with caution. Often times, the page might change, and locator will point to a completely different element from the one you expected. Instead, try to come up with a unique locator that will pass the [strictness criteria](#strictness).
+
 ## Locator vs ElementHandle
 
 :::caution
```

Comment:
suggestion to move locator vs elementHandle to handles doc so that it isnt something people read or find unless they look for it (as discussed with Pavel earlier)

Reply:
Sounds good.